php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2225,6 +2225,44 @@ class NDB_BVL_Instrument extends NDB_Page
         return $html;
     }
 
+    /**
+     * Return the HTML for the review section of the survey module
+     * By default, this will show every question from the parameter_type
+     * table for this instrument, but it can be overridden/customized by
+     * a instrument if that isn't the desired behaviour.
+     *
+     * @return string The HTML to be inserted into the review section of
+     *                the final page of the survey module for this instrument.
+     */
+    function getReactReview()
+    {
```

Comment:
Copy of the above function but submits back the resulting array before being passed to smarty